src/java/com/twitter/search/ingester/pipeline/util/BatchingClient.java

```diff
@@ -63,27 +63,25 @@ private void maybeBatchCall(RQ request) {
 
       // Make a copy of requests so we can modify it inside executeBatchCall without additional
       // synchronization.
-      frozenRequests = new HashSet<>(unsentRequests);
+      frozenRequests = Arrays.asList((RQ[]) unsentRequests.toArray());
```

Comment:
```suggestion
      frozenRequests = List.of(unsentRequests.toArray(new RQ[0]));
```
this better, pls commit the suggestion i want to upgrade my pea badge

Reply:
Does not compile.